src/js/media-chrome-listbox.js

```diff
@@ -183,6 +194,9 @@ class MediaChromeListbox extends window.HTMLElement {
     } else {
       item.setAttribute('aria-selected', 'true');
     }
+
```

Comment:
suggestion(non-blocking): we should check if the selection actually changed and only do the extra work (including dispatching the event) if the selection actually changed. Given the complexities of multiselectable support, this can be a followup if it feels like too heavy a lift rn.

Reply:
you mean to verify whether the new item is equivalent to the previous item and not trigger otherwise?

I'm not sure if that would be an issue since if you select an already selected item, it deselects it currently.